### PR TITLE
Fix design for Advanced Search modal

### LINKED ISSUE #893: Fix design for Advanced Search modal
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Screenshots of the issue:**

capture

### PR DESCRIPTION
**Pull request by jajodiaraghav**
Fixes #893 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Fixes the design
- Converts selected items to readonly
- Fixes the height of modal and adds a proper scrollbar

capture

### CODE REVIEW

File: src/app/advancedsearch/advancedsearch.component.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.menu-item-link {
+	font-size: 14px;
+}
+.modal-body {
```
Comment (harshit98) [inline src/app/advancedsearch/advancedsearch.component.css]: Leave 1 space.

File: src/app/advancedsearch/advancedsearch.component.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	max-height: 560px;
+	overflow-y: scroll;
+	overflow-x: hidden;
+}
```
Comment (harshit98) [inline src/app/advancedsearch/advancedsearch.component.css]: Leave space at EOF.

File: src/app/advancedsearch/advancedsearch.component.html [hunk 3]
```diff
@@ -4,39 +4,47 @@
...
+      <h2 class="modal-title" id="my-modal-label">Advanced Search</h2>
     </div>
 
     <div class="modal-body">
-      <ul class="nav nav-sidebar menugroup">
-        <li style="cursor:pointer;cursor: hand;" data-toggle="collapse" >
-          <h3>selected</h3>
+      <ul class="nav nav-sidebar menugroup" *ngIf="selectedelements.length">
+        <li data-toggle="collapse">
+          <h3 style="font-weight: bold">Applied Filters</h3>
```
Comment (harshit98) [inline src/app/advancedsearch/advancedsearch.component.html]: Avoid inline-styling.

**Comment by harshit98** [comment]
@jajodiaraghav The PR looks good to me. Some minor modifications have to be done. 

Can you check why Travis is failing? If you can fix it, kindly make a separate PR for it. 🎉

**Comment by jajodiaraghav** [comment]
@harshit98 I hae updated the code based on your comments. Please review

> Can you check why Travis is failing? If you can fix it, kindly make a separate PR for it.

The Travis is failing at master, not in the PR. I am not sure how to fix that.

> Also, please open up a separate issue for it and refer it here.

I am not sure what you mean by this. I have already opened an issue for this and the issue is already linked in the PR description.


Please review and merge.

**Comment by harshit98** [comment]
@jajodiaraghav Travis has been fixed. You can update this PR. 👍

**Comment by jajodiaraghav** [comment]
@harshit98 Updated. Please merge :)

**Pull request merged**